tsconfig.json

```diff
@@ -5,8 +5,19 @@
     "module": "commonjs",
     "esModuleInterop": true,
     "moduleResolution": "node",
-    "allowJs": true,
-    "checkJs": true
+    "allowJs": true, // Allow JavaScript files to be compiled.
```

Comment:
nit: JSON doesn't allow comments. I think TypeScript is lenient with this though.